karbon/karbon_cluster_registration_service.go

```diff
@@ -79,6 +79,45 @@ func (op ClusterRegistrationOperations) GetK8sRegistrationList(ctx context.Conte
 	return karbonClusterActionResponse, nil
 }
 
+func (op ClusterRegistrationOperations) GetK8sRegistrationListPage1(ctx context.Context) (*K8sClusterRegistrationListResponse, error) {
```

Comment:
These three seem to be methods with the same function signature with a different offset/page_size value set in the path. This should be a configurable parameter in the method signature and we can drop this down to a dingle function `GetK8sRegistrationList`.